applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_pw_line_integration_coefficients.cpp

```diff
@@ -0,0 +1,66 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//                   Gennady Markelov
+//
+
+#include "custom_elements/pw_line_integration_coefficients.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+
+#include <boost/numeric/ublas/assignment.hpp>
+#include <structural_mechanics_application_variables.h>
+
+using namespace Kratos;
+using namespace std::string_literals;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(PwLineIntegrationCoefficients_ReturnsCorrectValue, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Set
+    const auto pw_line_integration_coefficients = IntegrationCoefficientsCalculator{
+        std::make_unique<IntegrationCoefficientModifierForPwLineElement>()};
+    // The shape function values for this integration point are 0.2, 0.5 and 0.3 for nodes 1, 2 and 3 respectively
+    const Geometry<Node>::IntegrationPointType       integration_point(0.5, 0.3, 0.0, 0.5);
```

Comment:
Also here, I think we can get rid of some information (since it doesn't impact the test results, it helps to just declutter it a bit):
```suggestion
    const Geometry<Node>::IntegrationPointType       integration_point(0.0, 0.0, 0.0, 0.5);
```

Reply:
changed